server/store/subscriptions.go

```diff
@@ -198,3 +203,30 @@ func SubscriptionListFromJSON(bytes []byte) (*SubscriptionList, error) {
 	}
 	return subscriptionList, nil
 }
+
+func (s *Store) StoreSubscriptionChannelID(subscriptionID, webhookSecret, channelID string) error {
+	if err := s.StoreJSON(subscriptionID, SubscriptionWebhookSecretAndChannelMap{
+		webhookSecret: channelID,
```

Comment:
This reads a little weird, because it's not obvious that `StoreSubscriptionChannelID` is literally a map, and not a struct with a field named `webhookSecret`

Reply:
fixed